source/data/contributor.yaml

```diff
@@ -347,3 +347,8 @@
   linkedin: //www.linkedin.com/in/aleksandr-korolev-575045148
   drupal: //www.drupal.org/u/aleksandrkorolyov
   bio: Software Engineer | DrupalSquad
+- id: tmountjr
+  name: Tom Mount
+  avatar: //avatars.githubusercontent.com/u/1203675
+  github: //github.com/tmountjr
+  bio: Sr. Technical Engagement Manager
```

Comment:
@tmountjr feel free to adjust your ID, avatar, etc, and add other links if you want. Unfortunately this feature requires an externally hosted avatar image.